WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt

```diff
@@ -9,7 +9,7 @@ d_agonisticis = {
 	
 	capital = 712 # Carthage
 
-	title = "Holy Martyr"
+	title = "AGONISTIC_MARTYR"
```

Comment:
It will be displayed as AGONISTIC_MARTYR.
BTW  I also changed this title in my PR, replaced with HOLY_MARTYR key. Which one do you think is better?